tutorials/12-gaussian-process-latent-variable-model/12_gaussian-process-latent-variable-model.jmd

```diff
@@ -0,0 +1,271 @@
+---
+title: Gaussian Process Latent Variable Model
+permalink: /:collection/:name/
+redirect_from: tutorials/12-gaussian-process-latent-variable-model/
+---
+
+# Gaussian Process Latent Variable Model
+
+In a previous tutorial, we have discussed latent variable models, in particular probabilistic principal
```

Comment:
If you write one sentence per line, then this works much better with the diff. Also, the sentences are currently too long for the GitHub interface and for people who use more advanced editors with delete-sentence and similar shortcuts. For example, Visual Studio Code, Vim or Emacs.